modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/store/get_store_values.es.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import fetch from './../../util/fetch.es';
+import {isString} from 'metal';
+
+const TOKEN_SERIALIZE = 'serialize://';
+
+/**
+ * Gets the Store utility fetch values
+ * @param {!String} cmd
+ * @param keys
+ * @param callback
+ * @review
+ */
+
+export default function getStoreValues(cmd, keys, callback) {
+	cmd = 'getAll';
+
+	const doAsUserIdEncoded = Liferay.ThemeDisplay.getDoAsUserIdEncoded();
+
+	const url = `${Liferay.ThemeDisplay.getPathMain()}/portal/session_click?p_auth=${
+		Liferay.authToken
+	}${
+		doAsUserIdEncoded ? '&doAsUserId=' + doAsUserIdEncoded : ''
+	}&cmd=${cmd}&key=${keys}`;
```

Comment:
This code would not work, as we need to send keys as a json in body. see original code
```
				config.data.key = key;

				if (cmd == 'getAll') {
					config.dataType = 'json';
				}

				instance._ioRequest(config);
```

 We can greatly simplify the function: 
```
const formData = objectToFormData({
    cmd: 'getAll'
    p_auth: Liferay.authToken
});

formData.append('key', keys);

if (doAsUserIdEncoded) {
    formData.append('doAsUserId', doAsUserIdEncoded);
}
```